myriadeploy/get_logs.py

```diff
@@ -15,11 +15,10 @@ def get_std_logs_from_worker(hostname, dirname, username,
                              worker_id, description):
     mkdir_if_not_exists(description)
     if hostname == 'localhost':
-        uri = "%s/worker_%s_stdout" % (dirname, worker_id)
+        uri = "%s/worker_%s/worker_%s_stdout" % (dirname, worker_id, worker_id)
```

Comment:
Another convention to consider is `workers/1/` etc. This is less uniform WRT `master/`, but might make scenarios like running `grep -R` over all worker files simpler.